src/components/Navigation/Navigation.tsx

```diff
@@ -1,14 +1,13 @@
 import React, { FC } from 'react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Agree in general, though in this case Keeyan's other PR re-introduces the back button with conditional rendering rather than just commenting it out, so it's not a huge deal either way